### ISSUE TITLE
for games?

### ISSUE THREAD

**Issue by thelege2nd** [berkbavas/DORICalculator#1]
Hello, can this be used to calculate the FOV in any game?

**Comment by thelege2nd** [comment]
@berkbavas comments please.

**Comment by berkbavas** [comment]
Hi,

Sorry for the delay, I thought you were a robot.

Yes you can, provided that the perspective projection is used.

**Comment by thelege2nd** [comment]
> Hi,
> 
> Sorry for the delay, I thought you were a robot.
> 
> Yes you can, provided that the perspective projection is used.

no, I'm not a bot lol. But in case I wasn't clear, I want to calculate the FOV of a game FOV through screen.  (like pixel distances or angles) like through an image shown on the screen and use it for my FOV calculations? will this work or do I need to access the Game files itself?

**Comment by berkbavas** [comment]
Now the question is clear.
No, you cannot calculate it unless you know Cartesian distances between objects and Cartesian length of objects in advance. What you see on the screen is pixel distances whereas the perspective projection is applied and it is not easy by eye to estimate real distances.

Just search on the internet or hack game files.
